src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/EndpointService.java

```diff
@@ -111,7 +111,7 @@ public void deleteEndpoint(Long id) throws EndpointNotFoundException, ClientNotF
      * @throws IllegalGeneratedEndpointUpdateException  trying to update that is generated automatically
      * @throws IllegalArgumentException                 passing illegal combination of parameters
      */
-    public EndpointType updateEndpoint(Long id, EndpointPathAndMethod.MethodEnum method, String path)
+    public EndpointType updateEndpoint(Long id, EndpointUpdate.MethodEnum method, String path)
```

Comment:
OpenAPI models shouldn't be referenced in the service layer: https://confluence.niis.org/pages/viewpage.action?spaceKey=XRDDEV&title=API+backend+implementation

Maybe map the values to a separate Method enum?

Reply:
I decided to just pass string version of the method to service layer extra mapping feels a bit too much here.